tests/dummy/app/components/navigation-menu.hbs

```diff
@@ -8,7 +8,7 @@
         <LinkTo
           @route={{menuItem.route}}
           data-test-link={{menuItem.label}}
-          local-class="link {{if (eq menuItem.route this.rootRoute) "active-route"}}"
```

Comment:
By comparing `menuItem.route` to `rootRoute` (at least, in the way the getter used to be defined), we can land in a situation where _none_ of the menu items are highlighted (take on the `.active-route` class). For example, when the navigation menu includes links to nested routes.

This bug gave me another reason for using Ember's `.active` class instead.